charts/gardener/controlplane/charts/runtime/templates/apiserver/hvpa.yaml

```
@@ -0,0 +1,73 @@
+{{- if .Values.global.apiserver.hvpa.enabled }}
+{{- /* .Values.global.apiserver.replicaCount is of type 'float64', so let's cast it to string to have proper types for comparison */}}
+{{- if ne (.Values.global.apiserver.replicaCount | toString) "0" }}
+apiVersion: autoscaling.k8s.io/v1alpha1
+kind: Hvpa
+metadata:
+  name: gardner-apiserver-hvpa
+  namespace: garden
```

```suggestion
  namespace: garden
  labels:
    app: gardener
    role: apiserver
    chart: "{{ .Chart.Name }}-{{ .Chart.Version }}"
    release: "{{ .Release.Name }}"
    heritage: "{{ .Release.Service }}"
```
Can you please add the default labels to the hvpa?